### ISSUE TITLE
`balena create` command

### ISSUE THREAD

**Issue by zwhitchcox** [balena-io/balena-cli#2059]
In reference to https://github.com/balena-io/docs/issues/1506

It might be a good idea to have a `balena create` command, similar to the [`yarn create`](https://classic.yarnpkg.com/en/docs/cli/create/), except that any of the [getting started](https://www.balena.io/docs/learn/getting-started/raspberrypi4-64/nodejs/) languages/frameworks would be supported (e.g. Node, Python, C++, Rust, Go, and Haskell).

Basically, it could just clone the project and make some adjustments based on settings or maybe add some `balena-blocks` or something. I'm not sure what all functionality would be useful, but as a newcomer, I think I would personally find this useful.

**Comment by pdcastro** [comment]
@zwhitchcox, if you think this feature would greatly benefit balena users, get it discussed in a product meeting, ideally with more details about which programming languages, which files to include, where to store and maintain those files, who would maintain them (probably you!), whether it should be added to the web dashboard, and a rough idea of whether the feature is deemed to be a priority. Otherwise, by just creating this issue in the balena-cli repo, I don't see it getting any attention or traction... By the way, the repo we usually use for balena product features / ideas is https://github.com/balena-io/balena-io/issues (private).

**Comment by zwhitchcox** [comment]
Hey! thanks, I opened this a while ago, and I actually created one, but people didn't really seem interested, so I'm going to close this.
